benchmarks/setup/default/simple-starter.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
```

Comment:
:x: :warning: You need to adjust the makefile if you rename this file

Reply:
Oops, nice catch :+1: